specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-11-23/confidentialledger.json

```diff
@@ -0,0 +1,1021 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-23",
+    "title": "ConfidentialLedgerClient",
+    "description": "The ConfidentialLedgerClient writes and retrieves ledger entries against the Confidential Ledger service."
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
```

Comment:
I modified these fields explicitly because the CI was failing the linter check because of this (see [this rule](https://github.com/Azure/azure-openapi-validator/blob/main/docs/host-parameters-validation.md)). I'd check with the API board / reviewers if we need to change or suppress this rule somehow.

Reply:
@mikekistler @markweitzel @JeffreyRichter @johanste @lynshi Do you have any opinion on this? Should we suppress this rule or comply with the check?

Reply:
I'm not familiar with this rule at all. @johanste will have to answer

Reply:
I decided to revert this change back and use the same naming of the previous API versions for compatibility reasons. Hopefully, it should be fine to ignore the linter rule this time given the circumstances. Feel free to let me know if you have a different opinion or a better suggestion.

Reply:
IIRC, this rule was due to autorest and/or code generators making an assumption about the name of the endpoint parameter beinig `endpoint`.  Do we know if such an assumption remains?  The above alludes to it working with a different name.  If this requirement is no longer necessary then we should remove this linter rule.

Reply:
@lmazuel if this is ok with you can you please merge?

Reply:
Closing as no alternative solution was provided for fixing this CI check.